### ISSUE TITLE
A11y: Spatial Commerce - 3D file is not considered when navigating with keyboard

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5051]
Users can navigate through images using the keyboard seamlessly, but they are unable to access 3D files. Additionally, once on the 3D file, keyboard controls are not active, preventing users from interacting with the 3D content. The "Open AR View" button is also inaccessible via the keyboard. This presents significant accessibility (a11y) issues.

*Steps to Reproduce:*

# Open the page.
# Navigate through images by pressing the "Tab" key.
# Attempt to transition to the 3D file after the images. ===>It will skip the 3D file directly.
# Then click on the 3D file, try using keyboard controls to turn around or zoom in out.===> It will not work.
# Attempt to access the "Open AR View" button using the keyboard. ===> It will also skipped.

*Expected Result:*

* Users should be able to navigate to the 3D file using the keyboard.
* Users should have keyboard controls active to move the view with arrow keys and zoom in/out using the "+" and "-" keys.
* Users should be able to access the "Open AR View" button using the keyboard and activate it.

*Actual Result:*

* Users are unable to transition to the 3D file.
* Keyboard controls for moving and zooming within the 3D file are not functional.
* The "Open AR View" button is not accessible via keyboard navigation.

[^screen-capture (12).webm]

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-38805**



[created from NEXT-38805, comment 502032]

**Comment by shopwareBot** [comment]
[public] 

 > FixVersion Change: **6.6.9.0**

 This issue will be / was solved in Shopware version ***6.6.9.0***.



[created from NEXT-38805, comment 517622]

**Comment by shopwareBot** [comment]
[public] 

 > FixVersion Change: **6.6.10.0**

 This issue will be / was solved in Shopware version ***6.6.10.0***.



[created from NEXT-38805, comment 517624]
